server/game/cards/02_SHD/units/PrincipledOutlaw.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType, ZoneName } from '../../../core/Constants';
+
+export default class PrincipledOutlaw extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9040137775',
+            internalName: 'principled-outlaw'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'Choose a ground unit to exhaust',
```

Comment:
nit: our style for ability titles here would be `'Exhaust a ground unit'`. We try not to include "choose", "you may", etc. since that will be implied by the presence of the corresponding prompts